internal/plugins/sdk/api.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
This isn't _really_ a plugin, right? WDYT of moving the `sdk` package to `internal/pluginutil/sdk` or something similar?

When Phase 2 plugins exist, we could move the logic into a plugin at that point.